drivers/cyw43/cyw43.h

```diff
@@ -57,7 +57,7 @@ typedef struct _cyw43_t {
     volatile uint32_t wifi_scan_state;
     uint32_t wifi_join_state;
     void *wifi_scan_env;
-    int (*wifi_scan_cb)(void*, const cyw43_ev_scan_result_t*);
+    int (*wifi_scan_cb)(void *, const cyw43_ev_scan_result_t *);
```

Comment:
This file is currently not in the auto-format list, so let's not change any formatting here.